config/server.go

```diff
@@ -111,6 +111,14 @@ func GetServer(name string) *Server {
 	return nil
 }
 
+// GetServers - gets all the server names host has registered to.
+func GetServers() (servers []string) {
+	for _, server := range Servers {
+		servers = append(servers, server.Name)
+	}
+	return
```

Comment:
pointless return

Reply:
you need this empty return when you add a named return parameter

Reply:
oh right, didn't see the named return